sql-plugin/src/main/scala/com/nvidia/spark/rapids/SortUtils.scala

```diff
@@ -201,7 +201,23 @@ class GpuSorter(
           batches.foreach { cb =>
             tabs += GpuColumnVector.from(cb)
           }
-          Table.merge(tabs.toArray, cudfOrdering: _*)
+          // In the current version of cudf merge does not work for structs or lists (nested types)
+          // This should be fixed by https://github.com/rapidsai/cudf/issues/8050
```

Comment:
Should we file a follow-on in the plugin for us to revisit this when the cudf issue is done?

Reply:
#2252